### ISSUE TITLE
[Backend Developer] - Add Backend Framework

### ISSUE THREAD

**Issue by VincentJonathan** [kamranahmedse/roadmap.sh#168]
#### What roadmap is this issue about?

- [ ] Frontend Roadmap
- [x] Backend Roadmap
- [ ] DevOps Roadmap
- [ ] All Roadmaps

#### What is this issue about?

- [ ] Functionality of the website
- [ ] Discussion for a pull request I would want to open.
- [x] Addition of a new item
- [ ] Removal of some existing item
- [ ] Changing in arrangement
- [ ] General suggestion
- [ ] Sharing an Idea
- [ ] Something else

#### Please acknowledge the below listed

- [x] This is not a duplicate issue. I have searched and there is no existing issue for this.
- [x] I understand that these roadmaps are highly opinionated. The purpose is to not to include everything out there in these roadmaps but to have everything that is most relevant today comparing to the other options listed.
- [x] I have read the [contribution docs](../contributing) before opening this issue.

#### Enter the details about the issue here

Add web application frameworks for backend, few suggestions:

- Java
  - Spring / Spring Boot
- C#
  - ASP.NET
- PHP
  - Laravel
  - Slim
  - Lumen
- Javascript
  - express
- Python
  - Flask
  - Django
- Ruby
  - Ruby on Rails

**Comment by vildanzeynepkuyuk** [comment]
I agree with him.

**Comment by mraheelkhan** [comment]
For beginners, under the Php Codeigniter framework would be a plus.

**Comment by TimmJensen** [comment]
I agree with this aswell, it could sure use some popular framework as suggestions  :+1:

**Comment by patrykwozinski** [comment]
I think the PHP path without Symfony is no sense.
